docs/resources/README.md

```diff
@@ -0,0 +1,109 @@
+# Resources
+
+## Resource Detector
+
+OpenTelemetry .NET SDK provides a resource detector for detecting resource
+information from the `OTEL_RESOURCE_ATTRIBUTES` and `OTEL_SERVICE_NAME`
+environment variables.
+
+Custom resource detectors can be implemented:
+
+* ResourceDetectors should inherit from
+  `OpenTelemetry.Resources.IResourceDetector`, (which belongs to the
+  [OpenTelemetry](../../src/OpenTelemetry/README.md) package), and implement
+  the `Detect` method.
+
+A demo `ResourceDetector` is shown [here](../trace/extending-the-sdk/MyResourceDetector.cs):
```

Comment:
this can be moved to the common resources directory.

Reply:
Hey @cijothomas, thank you for reviewing it again 🙂
I've just created a [new project for resources](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5188/commits/a3d3c5d00a17f3976cd2f60b328d4774b7014c0f) with all the code snippets that I mentioned in the doc.